src/index.html

```diff
@@ -17,16 +17,32 @@
     />
   </head>
   <body>
-    <input
-      type="text"
-      data-qa="keypress"
-      placeholder="Try “Los Angeles“"
-    />
-
-    <input
-      type="text"
-      data-qa="hover"
-      placeholder="Try “Los Angeles“"
-    />
+    <form
+      action="#"
+      method="post"
+      data-qa="big"
+    >
+      <label class="search-bar__label search-bar__label--big"></label>
+      <input
+        type="search"
+        data-qa="keypress"
+        placeholder="Try “Los Angeles“"
+        class="search-bar__input search-bar__input--big"
+      />
+    </form>
+    <form
+      action="#"
+      method="post"
+      data-qa="small"
+    >
+      <label class="search-bar__label search-bar__label--small"></label>
+      <input
+        name="searchSmall"
```

Comment:
why did you add name here? use the same style naming (if you want to add this)

Reply:
Delete name="searchSmall". I forgot delete it. Sorry)